quizapp/src/main/java/billennium/quizapp/controller/CandidateController.java

```diff
@@ -34,4 +39,9 @@ public String getUserId(@PathVariable("email") String email) {
     public Candidate saveUser(@RequestBody CandidateEmailDto candidateEmailDto) {
         return candidateService.save(candidateEmailDto.getEmail());
     }
+
+    @GetMapping(GET_CANDIDATE + ID_PARAM)
+    public CandidateDto getCandidate(@PathVariable String id) {
+        return candidateService.getCandidateDto(UUID.fromString(id));
```

Comment:
`getById`. już zwracałem kilka razy uwage na nazwenictwo. jeżeli coś jest z serwisu np Candidate to w metododzie nie musi być słowa Candidate bo się powtarza.

 `@PathVariable String id` <- String ?